articles/quickstarts/update.md

```diff
@@ -212,7 +212,7 @@ The update procedure depends on whether you originally installed using conda or
 
     ```
     Name: qsharp
-    Version: 0.12.20070124
+    Version: 0.12.20072018
```

Comment:
Actually this was already incorrect, the Python package version is still using the four-part number.
```suggestion
    Version: 0.12.2007.2018
```